react/src/pages/Home.js

```diff
@@ -1,20 +1,13 @@
 import Navbar from '../components/Navbar';
 import Header from '../components/Header';
+import Boxes from '../components/Boxes';
```

Comment:
Jeśli chciałbyś się nauczyć czegoś fajnego, to możesz spróbować ogarnąć absolute paths w tym projekcie. Podejrzewam, że będziesz musiał do tego wykorzystać "craco", żeby dodać kilka linijek konfiguracji do webpacka. Wtedy można zrobić np. `import Boxes from '@/components` zamiast podawać relatywną ścieżkę do pliku. Teraz to nie robi większej różnicy, ale jakbysmy kiedys mieli cos takiego:
`import { getUser } from '../../../../../helpers'`
to możemy z tego zrobić...
`import { getUser } from '@/helpers'`

Pomocny stuff:

https://levelup.gitconnected.com/understand-and-configure-absolute-import-paths-in-javascript-5cde3be2630d (absolute paths)
https://www.npmjs.com/package/@craco/craco (craco, ktore pozwala na konfiguracje webpacka w bezpieczniejszy sposob niz `react eject`

W razie pytań jestem dostępny do piątku